pkgs/default.nix

```diff
@@ -90,6 +92,7 @@ pkgs: oldpkgs: {
   #     };
   #   }
   #
+  
```

Comment:
Please remove this newline to be consistent with the other function comments in this file.